docs/insomnia/version-control-sync.md

```diff
@@ -92,12 +91,11 @@ Branches can be useful for many reasons, but the two main use cases are the foll
 Branches are managed within the branches dialog. Here, you can create local branches, merge branches, and fetch remote branches.
 
 ### Share a Request Collection
-Request Collections can be shared with a team via the main sync menu. To get started sharing workspaces with your team, see Team Collaboration.
 
-### Revert to a Previous Snapshot
+Request Collections can be shared with a team via the branch dropdown menu. To get started sharing workspaces with your team, see [Team Collaboration](/insomnia/team-collaboration).
```

Comment:
The first part is no longer valid. A user can't choose which team to share a collection with (you could previously change this in the app, but can't anymore). The contents in the team-collaboration article are valid though, so I think just link off to that.